src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -8218,4 +8218,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_InterpolatedStringHandlerArgumentDisallowed" xml:space="preserve">
     <value>Interpolated string handler arguments are not allowed in this context.</value>
   </data>
+  <data name="ERR_ExtensionBlockCollision" xml:space="preserve">
+    <value>This extension block collides with another extension block. They have different signatures, but result in the same marker type identifier.</value>
```

Comment:
>marker type [](http://example.com/codeflow?start=126&length=11)

I think the reference to "marker type" is going to be confusing, it is not part of the language